typings/vscode-mssql.d.ts

```diff
@@ -433,6 +438,297 @@ declare module 'vscode-mssql' {
 		validateStreamingJob(packageFilePath: string, createStreamingJobTsql: string): Thenable<ValidateStreamingJobResult>;
 	}
 
+	/**
+	 * Interface for working with .sqlproj files
+	 */
+	export interface ISqlProjectsService {
```

Comment:
I wonder if at some point it would be easier to split the sql proj typings off into their own typings file just to keep everything related in one place.

Reply:
Yeah... I start getting an itch to refactor a file around 1k lines, and we're certainly hitting that here.